### PR TITLE
Issue/s2e 209 savevm on reboot option

### PR DESCRIPTION
**Pull request by adrianherrera**

### CODE REVIEW

File: savevm.c [hunk 1]
```diff
@@ -1797,6 +1799,11 @@ int qemu_savevm_state_complete(QEMUFile *f)
 	if (se->save_state == NULL && se->vmsd == NULL)
 	    continue;
 
+    if (dont_save_slirp && !strcmp(se->idstr, "slirp")) {
```
Comment (vitalych) [inline savevm.c]: indentation looks odd

File: slirp/tcp_input.c [hunk 2]
```diff
@@ -354,7 +344,18 @@ tcp_input(struct mbuf *m, int iphlen, struct socket *inso)
 	 * the only flag set, then create a session, mark it
 	 * as if it was LISTENING, and continue...
 	 */
-        if (so == NULL) {
+    if (so == NULL) {
```
Comment (vitalych) [inline slirp/tcp_input.c]: Indentation

**Comment by vitalych** [comment]
lgtm. Some small issues:
- indentation
- "ported" in commit msg, replace it with "added" (users would wonder where it was ported from)

**Comment by adrianherrera** [comment]
Ok, all fixed!

File: slirp/tcp_input.c (hunk 2 above)
Comment (adrianherrera) [reply slirp/tcp_input.c]: ahhh I see, it's a tab

**Review by vitalych — APPROVED** [review]

**Pull request merged**